app/renderer/components/Inspector/Source.js

```diff
@@ -10,6 +10,11 @@ const IMPORTANT_ATTRS = [
   'resource-id',
   'AXDescription',
   'AXIdentifier',
+  'text',
+  'label',
+  'value',
+  'id',
+  'class',
```

Comment:
I don't think `class` should be always shown in the XML view. On Android (which I would expect is the most common use-case for the inspector) this just adds a duplicate of the element type, needlessly bloating every single element item.
At the same time, it would be great if it _was_ shown in webview mode.

Reply:
Hmm yeah good point. I added it thinking of webviews. We probably need to refactor this code so that all of these arrays are sensitive to which platform we're dealing with.

And this could be part of a bigger project that lets drivers declare facts about themselves to the inspector so that we don't have to hardcode these things in the inspector itself.

I've got another open PR, I'll remove class from that one so at least we keep the status quo for now.